index.js

```diff
@@ -167,8 +168,8 @@ export const refresh = (
 };
 
 export const revoke = async (
-  { clientId, issuer, serviceConfiguration },
-  { tokenToRevoke, sendClientId = false }
+  { clientId, issuer, serviceConfiguration, clientSecret },
+  { tokenToRevoke, sendClientId = false, includeBasicAuthorization = false }
```

Comment:
`includeBasicAuthorization` should also be added to `RevokeConfiguration` in `index.d.ts`

Reply:
please check a22518e9c1044b6522d0ccf8d0ceba4eee4d2884